atomate/vasp/firetasks/lobster_tasks.py

```diff
@@ -0,0 +1,349 @@
+                       "GROSSPOP.lobster", "ICOHPLIST.lobster", "ICOOPLIST.lobster", "lobster.out",
+                       "projectionData.lobster"]
+
+VASP_OUTPUT_FILES = ["OUTCAR", "vasprun.xml", "CHG", "CHGCAR", "CONTCAR", "INCAR", "KPOINTS", "POSCAR", "POTCAR",
+                     "DOSCAR", "EIGENVAL", "IBZKPT", "OSZICAR", "PCDAT", "PROCAR", "REPORT", "WAVECAR", "XDATCAR"]
+VASP_OUTPUT_FILES_without_WAVECAR = ["OUTCAR", "vasprun.xml", "CHG", "CHGCAR", "CONTCAR", "INCAR", "KPOINTS", "POSCAR",
+                                     "POTCAR",
+                                     "DOSCAR", "EIGENVAL", "IBZKPT", "OSZICAR", "PCDAT", "PROCAR", "REPORT", "XDATCAR"]
+
+MODULE_DIR = os.path.dirname(os.path.abspath(__file__))
+logger = logging.getLogger(__name__)
+
+
+@explicit_serialize
+class WriteLobsterinputfromIO(FiretaskBase):
+    """
+    will write lobsterin from POSCAR, INCAR, POTCAR
+    Required Params:
+        poscar_path (str): path of POSCAR
+        incar_path (str): path of INCAR
+        potcar_path (str): address to POSCAR
+        option (str): options as in Lobsterin.standard_calculations_from_vasp_files
+    Optional Params:
+        user_supplied_basis (dict): dictionary including the basis for each atom type
+        user_lobsterin_settings (dict): dictionary that will be used to overwrite settings in Lobsterin dict
+    """
+    required_params = ["poscar_path", "incar_path", "potcar_path", "option"]
+    optional_params = ["user_supplied_basis", "user_lobsterin_settings"]
+
+    def run_task(self, fw_spec):
+        poscar_path = self.get("poscar_path", "POSCAR")
+        incar_path = self.get("incar_path", "INCAR")
+        potcar_path = self.get("potcar_path", "POTCAR")
+        option = self.get("option", "standard")
+        user_supplied_basis = self.get("user_supplied_basis", None)
+        if user_supplied_basis is None:
+            lobsterinput = Lobsterin.standard_calculations_from_vasp_files(poscar_path, incar_path, potcar_path,
+                                                                           option=option)
+        else:
+            lobsterinput = Lobsterin.standard_calculations_from_vasp_files(poscar_path, incar_path, None, option=option,
+                                                                           dict_for_basis=user_supplied_basis)
+        additional_input = self.get("user_lobsterin_settings", None)
+        if additional_input:
+            for key, parameter in additional_input.items():
+                lobsterinput[key] = parameter
+
+        lobsterinput.write_lobsterin("lobsterin")
+
+
+@explicit_serialize
+class RunLobster(FiretaskBase):
+    """
+    Starts the Lobster Job
+    Optional params:
+        lobster_cmd (str): command to run lobster
+        gzip_output (bool): If true, output (except WAVECAR) will be gzipped.
+        gzip_WAVECAR (bool): If true, WAVECAR will be gzipped

```

Comment:
Will be added. Will also set the default for gzip_output to True. 

Btw, I had to do this because when custodian is used, all outputs from the queue managing system will be zipped after this firetaks and the rest of the logging will not appear in the files. If you have a better solution for this, I am happy to adapt this in the future.